stock_location_limit_product/models/stock_location_limit.py

```diff
@@ -0,0 +1,41 @@
+# Copyright (C) 2019 Open Source Integrators
+# Copyright (C) 2019 Serpent Consulting Services Pvt. Ltd.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html)
+
+from odoo import api, fields, models
+
+
+class StockLocationLimit(models.Model):
+    _name = 'stock.location.limit'
+    _description = 'Stock Location Limit Product'
+    _rec_name = 'product_id'
+
+    _sql_constraints = [
```

Comment:
You have to take into account the 'active' field.

To do such constrain, see this : https://github.com/OCA/partner-contact/blob/10.0/partner_bank_sort_code/models/res_bank.py#L16

Reply:
hello @rousseldenis, can you suggest to me the reference for  EXCLUDE constraint for the relational table? I checked this reference but as per my knowledge, we can only use the current table's column in the constraint.

Reply:
Mhh, maybe my remark has no sense as if a product is not active anymore, there is no reason to create stock location limit.

Reply:
yes  :) so can I add simple uniqe constrain as before?